ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py

```diff
@@ -3,11 +3,21 @@
 
 
 def argsort(
-    x,
-    /,
-    *,
-    axis=-1,
-    kind=None,
-    order=None,
+        x,
+        /,
+        *,
+        axis=-1,
+        kind=None,
+        order=None,
 ):
     return ivy.argsort(x, axis=axis)
+
+
+def argmax(x,
+           /,
+           *,
+           axis=None,
+           keepdims=False,
+           out=None,
+           ):
+    return ivy.argmax(x, axis=axis)
```

Comment:
You should pass in the `keepdims` and `out` parameter here too.

Reply:
I noticed that I wrote the function and the test in the wrong files, according to the ivy documentation, "argmax" is in the "searching" category. I will fix the mistake I made.